.github/workflows/cd.yml

```diff
@@ -0,0 +1,59 @@
+name: CD
+
+on:
+  push:
+    branches: [ "master" ]
+    tags: [ "*" ]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+
+      - name: Setup dependencies
+        run: npm install
+
+      - name: Build xsnippet-web
+        run: npm run build
+
+      - name: Create xsnippet-web.tar.gz
+        run: tar cvzf xsnippet-web.tar.gz -C dist/ .
+
+      - name: Setup release metadata
+        run: |
+          PRERELEASE=true
+          VERSION=nightly
+
+          if [[ $GITHUB_REF =~ ^refs/tags/.* ]]; then
+            if [[ $GITHUB_REF =~ ^refs/tags/\d+.\d+.\d+$ ]]; then
```

Comment:
Shouldn't this be `^refs/tags/\d+\.\d+\.\d+$`? I think `.` matches any single character, while `\.` matches the dot character specifically?

Reply:
Thanks! What's worse is that `\d` does not work in bash. :thinking:  I tested this script manually and replaced `\d` with `[0-9]` groups. I also adjusted a bit the regular expression to take into account `1.0` versions (i.e. without patch version).